org.learningequality.Kolibri.Content.json.in

```diff
@@ -0,0 +1,25 @@
+{
+  "id": "org.learningequality.Kolibri.Content.@ID@",
+  "branch": "@BRANCH@",
+  "build-extension": true,
+  "runtime": "org.learningequality.Kolibri",
+  "runtime-version": "master",
+  "sdk": "org.freedesktop.Sdk//19.08",
+  "appstream-compose": false,
```

Comment:
Would having some appdata mean it is displayed in GNOME Software?

Reply:
I'm really not sure how that part should work, probably yes, we need an AppData template to fill out.